ui/src/app/pages/signed-in/component.ts

```diff
@@ -114,11 +114,18 @@ export class SignedInComponent implements OnInit, OnDestroy, AfterViewInit {
       this.minimizeChrome = minimizeChrome;
     }));
 
+    const lastActive = window.localStorage.getItem(INACTIVITY_CONFIG.LOCAL_STORAGE_KEY_LAST_ACTIVE);
```

Comment:
I move this from the root component file into the signed-in component since we only log out the user if they are signed out anyways.